src/sqlancer/clickhouse/gen/ClickHouseExpressionGenerator.java

```diff
@@ -380,7 +380,7 @@ public ClickHouseSelect generateSelect() {
     @Override
     public List<ClickHouseJoin> getRandomJoinClauses() {
         List<ClickHouseExpression.ClickHouseJoin> joinStatements = new ArrayList<>();
-        if (globalState.getClickHouseOptions().testJoins && Randomly.getBoolean()) {
```

Comment:
Orignal logic was meant to not test joins at all when this setting is disabled. But I do not remember why I did it this way. Maybe it is correct to leave it as is, but change default setting to `enabled`?